cli/appsecuritykey.go

```diff
@@ -0,0 +1,60 @@
+package cli
+
+import (
+	"database/sql"
+	"fmt"
+
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/cli/clibase"
+	"github.com/coder/coder/coderd/database"
+	"github.com/coder/coder/coderd/database/migrations"
+)
+
+func (*RootCmd) appSecurityKey() *clibase.Cmd {
+	var postgresURL string
+
+	root := &clibase.Cmd{
+		Use:   "app-security-key",
```

Comment:
We should change this so the moon fetches the key on startup. That can be part of the registration issue though.

Reply:
Agreed, removed this cli cmd.